spec/spec.md

```diff
@@ -839,6 +844,80 @@ const ERROR_ATTEST_ORIGIN_ANDROID = -100
 {% include_relative _diagrams/reauthorize_and_sign.plantuml %}
 {% endplantuml %}
 
+### Deprecated Methods
+
+These methods have been deprecated in the current version of the specification, but should still be supported by wallet endpoints for backwards compatibility. These methods may be removed in a future version of the specification. 
```

Comment:
Rather than repeat the entire message here, WDYT about linking to an archived v1.0 of the spec? It'd also be in GitHub, so no risk of it disappearing on us. It'll help keep this spec concise.

Reply:
yeah I like that. My plan was to maintain a spec1.0.md file and make a spec2.0.md. I may do that in a sperate PR tho (for review purposes I prefer the file diff here on spec.md)